submissions/OgeEjim/html-forms/ContactForm.html

```diff
@@ -5,6 +5,10 @@
    <title>Enquires</title>
 </head>
 <body>
+    <div>
+        <a href="ContactForm.html">Contact Form</a>
+        <a href="productpageforcinnamoncakes.html">Product</a>
```

Comment:
This is wrongly linked, the `productpageforcinnamoncakes.html` is inside your `HTML` folder. doing it this `href="productpageforcinnamoncakes.html"` way won't target that file. So kindly correct yourself. Start from the folder name and go a step up

Reply:
You should not copy the `productpageforcinnamoncakes.html` from your `HTML` folder to `html-forms` folder

Reply:
> You should not copy the `productpageforcinnamoncakes.html` from your `HTML` folder to `html-forms` folder

good morning ma, I have made the corrections requested